include/exadg/operators/operator_base.cpp

```diff
@@ -2153,6 +2158,176 @@ OperatorBase<dim, Number, n_components>::evaluate_face_integrals() const
          (integrator_flags.face_integrate != dealii::EvaluationFlags::nothing);
 }
 
+template<int dim, typename Number, int n_components>
+void
+OperatorBase<dim, Number, n_components>::compute_factorized_additive_schwarz_matrices() const
+{
+#ifdef DEAL_II_WITH_TRILINOS
+  internal_compute_factorized_additive_schwarz_matrices<dealii::TrilinosWrappers::SparseMatrix>();
+#elif DEAL_II_WITH_PETSC
+  internal_compute_factorized_additive_schwarz_matrices<dealii::PETScWrappers::MPI::SparseMatrix>();
+#else
```

Comment:
When working on PR #681, I realized that in other places of ExaDG we want to be able to compile for both Trilinos and PETSc and then decide which code to use depending on a parameter (`SparseMatrixType` introduced in #681, `AMGType` already used in AMG preconditioners). So I think we should adapt this code to be consistent with other parts of ExaDG.